docs/static/settings/configuration-management-settings.asciidoc

```diff
@@ -78,3 +78,19 @@ the client’s certificate.
 `xpack.management.elasticsearch.ssl.keystore.password`::
 
 Optional setting that provides the password to the keystore.
+
+`xpack.management.elasticsearch.cloud_id`::
+
+If you're using Elasticsearch cloud, you should specify the identifier here.
```

Comment:
I don't _think_ we refer to it as `Elasticsearch cloud`, but rather as `Elastic Cloud`, so maybe:

```suggestion
If you're using {es} in Elastic Cloud, you should specify the identifier here.
```

Reply:
You're on the right track!  Should be `{es} in {ecloud}`
I'm doing a quick local test to be sure that renders expected results.

Reply:
^^ We have attributes in place that sub in the correct product names.  It's good for ensuring consistency and handy in case we rename or rebrand.